hack/verify-code-patterns.sh

```diff
@@ -22,3 +22,10 @@ if [[ -n "$out" ]]; then
   echo >&2 "$out"
   exit 1
 fi
+
+out="$(grep --include '*.go' --exclude "*_test.go" --exclude 'constants.go' --exclude-dir 'vendor/' -EIrn '\.yaml"' || true)"
```

Comment:
this is a long line now, let's split into multiple with `\`

also `--exclude 'vendor/*'` should be working here.

Reply:
I improved the legibility as suggested.

For my GNU sed, the `--exclude 'vendor/*'` command does not work. But I think the `--exclude-dir` version also works for BSD sed. At least https://www.freebsd.org/cgi/man.cgi?query=grep&sektion=&n=1 sounds alike.

Reply:
You're totally right, something’s up with this --exclude.
I think we can keep this.